src/constants/zapperServiceConstants.js

```diff
@@ -0,0 +1,93 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2019 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+export const PROTOCOLS = {
+  AUTO_FARM: 'autofarm',
+  AAVE: 'aave',
+  AAVE_AMM: 'aave-amm',
+  AAVE_V2: 'aave-v2',
+  ALCHEMIX: 'alchemix',
+  ALPHA: 'alpha',
+  B_PROTOCOL: 'b-protocol',
+  BADGER: 'badger',
+  BALANCER: 'balancer',
+  BANCOR: 'bancor',
+  BARNBRIDGE: 'barnbridge',
+  BEEFY: 'beefy',
+  BELLA: 'bella',
+  BITCOIN: 'bitcoin',
+  BZX: 'bzx',
+  COMPOUND: 'compound',
+  COVER: 'cover',
+  CREAM: 'cream',
+  CURVE: 'curve',
+  DEFISAVER: 'defisaver',
+  DEFIVADEX: 'derivadex',
+  DEVERSIFI: 'deversifi',
+  DHEDGE: 'dhedge',
+  DFORCE: 'dforce',
+  DODO: 'dodo',
+  DSD: 'dsd',
+  DYDX: 'dydx',
+  ELLIPSIS: 'ellipsis',
+  ESD: 'esd',
+  FUTURESWAP: 'futureswap',
+  IDLE: 'idle',
+  HARVEST: 'harvest',
+  HEGIC: 'hegic',
+  KEEPER_DAO: 'keeper-dao',
+  LINKSWAP: 'linkswap',
+  LOOPRING: 'loopring',
+  LIQUITY: 'liquity',
+  MAKER: 'maker',
+  MOONISWAP: 'mooniswap',
+  MUSHROOM: 'mushroom',
+  ONEINCH: '1inch',
+  PANCAKESWAP: 'pancakeswap',
+  NFT: 'nft',
+  OTHER: 'other',
+  PICKLE: 'pickle',
+  POOLTOGETHER: 'pooltogether',
+  QUICKSWAP: 'quickswap',
+  RARI: 'rari',
+  REALT: 'realt',
+  REFLEXER: 'reflexer',
+  RIBBON: 'ribbon',
+  SADDLE: 'saddle',
+  SFINANCE: 'sfinance',
+  SHELL: 'shell',
+  SMOOTHY: 'smoothy',
+  SNOWSWAP: 'snowswap',
+  SPOOKYSWAP: 'spookyswap',
+  SUSHISWAP: 'sushiswap',
+  SWERVE: 'swerve',
+  SYNTHETIX: 'synthetix',
+  TOKENSETS: 'tokensets',
+  TOKENS: 'tokens',
+  UNISWAP: 'uniswap',
+  UNISWAP_V2: 'uniswap-v2',
+  UNISWAP_V3: 'uniswap-v3',
+  UNIT: 'unit',
+  VALUE: 'VALUE',
```

Comment:
Sorry did i caps this? I think it's supposed to be lower case...

```suggestion
  VALUE: 'value',
```